compiler/luci/import/include/luci/Import/Nodes/CircleVariable.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_IMPORT_OP_CIRCLE_VARIABLE_H__
+#define __LUCI_IMPORT_OP_CIRCLE_VARIABLE_H__
+
+#include "luci/Import/GraphBuilderContext.h"
+
+#include <luci/IR/Nodes/CircleVariable.h>
+
+/*
+ * @note  Circle does not have node for variable tensor
+ *        Methods here provide helper that creates CircleVariable from
+ *        Tensor having is_variable true value.
+ */
+
+namespace luci
+{
+
+CircleVariable *create_circlevariable(GraphBuilderContext *context, int32_t tensor_index);
```

Comment:
Can I ask you why `CircleVariable` not used `TypedNodeBuilder<NodeBuilderType::BUFFER>` contrast to `CircleConst`?
`CircleVariable` seems quite similar with `CircleConst` except only `is_variable`.
Therefore, for me, it would be better to use similar design with `CircleConst` for better understanding and readibility.
But I want to know if you intended something else :)

Reply:
That is a choice what I didn't select it for now; there is no need as of now.
If someone has a requiremment then he/she can do it.

Reply:
Hmm...  okay.
FYI, since all other nodes in `luci/Import/Nodes/*.h` use class which is not function, this would break the unification.

Reply:
> use class which is not function, this would break the unification.

Um? Did we have requirements to provide `unification` ? I designed this but I don't remember putting this in here.

Reply:
My questions was because of following thoughts

- When we add a new CircleNode, we followed the design of existing nodes (kind of copy&paste)
- When we add `CircleVariable`, design is different with the existing nodes

So I wondered why they are different, not objected...!

Reply:
> When we add CircleVariable, design is different with the existing nodes

CircleVariable is to represent circle ensors(or from tflite tensor) having true with `is_variable`. As like CircleConst, which was introduced from the first place, to represent tensors with initial values. (CircleVariable doesn't have initial values, from the BUFFER, as of now).
Preparing(or initializing) CircleVariable followed procedures from CircleConst, which was first created with a method (not a class). Recently it was changed using a class like the Op nodes in purpose of #8042 (if you didn't notice).
As I wrote currently there is no need to prepare CircleVariable through a builder.
So I wrote it using plain method, which follow the initial implementation (copy-paste).